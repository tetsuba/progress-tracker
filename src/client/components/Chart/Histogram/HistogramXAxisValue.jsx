import React from 'react'

type Props = {
  letter: string,
}

export default function HistogramXAxisValue(props: Props) {
  const { letter } = props
  const styles = {
    width: '3%',
    textAlign: 'center',
  }

  return <div style={styles}>{letter}</div>
}
