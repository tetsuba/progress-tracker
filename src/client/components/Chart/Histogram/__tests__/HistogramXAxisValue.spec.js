import { testRenderer } from '../../../../../test/testHelper'
import HistogramXAxisValue from '../HistogramXAxisValue'

describe('<HistogramXAxisValue>', () => {
  describe('@Render', () => {
    it('Should render a value', () => {
      const props = {
        letter: 'a',
      }
      const wrapper = testRenderer(HistogramXAxisValue, props)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
