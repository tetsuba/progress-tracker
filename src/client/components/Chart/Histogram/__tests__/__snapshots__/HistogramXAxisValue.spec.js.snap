// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HistogramXAxisValue> @Render Should render a value 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
      "width": "3%",
    }
  }
>
  a
</div>
`;
